please_login.php

```diff
@@ -92,7 +92,7 @@
 <div class='paper'>
 <div class='title'>Please log in</div>
 <p>You have to log in to play Triniate.</p>
-<p style='text-align: center; position: relative;'><input type='button' value='To login page' oNclick="window.location='<?php echo $triniate_playpage;?>';"></p>
+<p style='text-align: center; position: relative;'><input type='button' value='To login page' oNclick="window.location='<?php /* Removed */ echo "";?>';"></p>

```

Comment:
Why this change?

Reply:
It wasn't working as expected and I was going to change it to work. Maybe try adding `global $triniate_playpage` into `github_settings.php`?